src/app.js

```diff
@@ -68,7 +68,7 @@ app.use((req, res, next) => {
       scriptSrc: ["'self'", '*.googletagmanager.com'],
       scriptSrcElem: ["'self'", 'https://*.googletagmanager.com', `'nonce-${res.locals.nonce}'`],
       imgSrc: ["'self'", 'data:', 'www.googletagmanager.com', '*.google-analytics.com'],
-      connectSrc: ["'self'", '*.google-analytics.com', '*.analytics.google.com', '*.googletagmanager.com'],
+      connectSrc: ["'self'", '*.google-analytics.com', '*.analytics.google.com', '*.googletagmanager.com', 'https://kit.fontawesome.com', 'https://ka-p.fontawesome.com'],
```

Comment:
Is this to remove a warning/error in the console?

Reply:
good catch, we actually don't need these, I will remove